api/DocumentFragment.json

```
@@ -77,7 +77,7 @@
               "version_added": true
             },
             "safari": {
-              "version_added": false
+              "version_added": true
             },
             "safari_ios": {
               "version_added": null
```

Coincidentally, this would also be the same version number for iOS:

```suggestion
              "version_added": "8"
```